graphql/graphql.d.ts

```diff
@@ -1552,36 +1468,36 @@ declare module "graphql/type/definition" {
     class GraphQLInputObjectType {
         name: string;
         description: string;
-        constructor(config: InputObjectConfig);
-        getFields(): InputObjectFieldMap;
+        constructor(config: GraphQLInputObjectConfig);
+        getFields(): GraphQLInputObjectFieldMap;
         toString(): string;
     }
 
-    interface InputObjectConfig {
+    export interface GraphQLInputObjectConfig {
         name: string;
-        fields: Thunk<InputObjectConfigFieldMap>;
+        fields: Thunk<GraphQLInputObjectConfigFieldMap>;
         description?: string;
     }
 
-    interface InputObjectFieldConfig {
+    export interface GraphQLInputObjectFieldConfig {
         type: GraphQLInputType;
         defaultValue?: any;
         description?: string;
     }
 
-    interface InputObjectConfigFieldMap {
-        [fieldName: string]: InputObjectFieldConfig;
+    export interface GraphQLInputObjectConfigFieldMap {
+        [fieldName: string]: GraphQLInputObjectFieldConfig;
     }
 
-    interface InputObjectField {
+    export interface GraphQLInputObjectField {
         name: string;
         type: GraphQLInputType;
         defaultValue?: any;
         description?: string;
     }
 
-    interface InputObjectFieldMap {
-        [fieldName: string]: InputObjectField;
+    export interface GraphQLInputObjectFieldMap {
```

Comment:
I’d argue that in the interest of keeping type definition quality high, we don’t need to match the source code exactly. We should definitely need to exactly match functionality, but in the names of type aliases we have a bit more freedom to improve upon the source code itself.

I’d also argue that the intention for these types is to be named with a ‘GraphQL’ prefix given the other types and the names probably were meant to align with the `GraphQLObjectType` types.

I’ve opened a PR here to change the source code https://github.com/graphql/graphql-js/pull/526, we’ll see how that goes. In the meantime I can change these back if you’d like :+1: